src/Files.App/ViewModels/Settings/FoldersViewModel.cs

```diff
@@ -329,8 +345,9 @@ public int SelectedDefaultGroupingIndex
 					OnPropertyChanged(nameof(SelectedDefaultGroupingIndex));
 
 					UserSettingsService.FoldersSettingsService.DefaultGroupOption = value == FileTagGroupingIndex ? GroupOption.FileTag : (GroupOption)value;
-					// Raise an event for the 'Group in descending order' toggle switch availability
-					OnPropertyChanged(nameof(isDefaultGrouped));
+					// Raise an event for the grouping option toggle switches availability
+					OnPropertyChanged(nameof(IsDefaultGrouped));
+					OnPropertyChanged(nameof(IsGroupByDate));
```

Comment:
Add a line before a comment line.

```suggestion

					// Raise an event for the grouping option toggle switches availability
					OnPropertyChanged(nameof(IsDefaultGrouped));
					OnPropertyChanged(nameof(IsGroupByDate));
```